nautobot/docs/rest-api/overview.md

```diff
@@ -785,3 +785,478 @@ http://nautobot/api/dcim/sites/ \
 
 !!! note
     The bulk deletion of objects is an all-or-none operation, meaning that if Nautobot fails to delete any of the specified objects (e.g. due a dependency by a related object), the entire operation will be aborted and none of the objects will be deleted.
+
+### Depth Query Parameter
```

Comment:
Also need to remove the `Brief Format` section earlier in this document and add a note that it was removed in 2.0.0; may be other changes needed in this document as well.